loadbalancer/lb_rule_condition.go

```diff
@@ -13,8 +13,47 @@ package loadbalancer
 type LbRuleCondition struct {
 
 	// A flag to indicate whether reverse the match result of this condition
-	Inverse bool `json:"inverse"`
+	Inverse bool `json:"inverse,omitempty"`
```

Comment:
What about all the booleans omitempty here?

Reply:
Good point. Luckily for us inverse and case sensitive here are defaulted to false. But we need a proper solution to the omitempty issue.